sources/web/datalab/static/appbar.html

```diff
@@ -75,6 +75,14 @@
           </div>
           <button id="stopVmButton" title="Stop VM" class="btn btn-danger" style="width: 100%; float: none">Stop VM</button>
         </div>
+        <li><a id="idleTimeoutEnabledButton" href="#" style="display:none;">
```

Comment:
we should add `title` attributes for both these buttons, so the user has proper expectation before clicking